include/hlslib/common/OpenCL.h

```diff
@@ -785,8 +721,7 @@ class Buffer {
     CopyToDevice(offsetSource, numElements, other, 0);
   }
 
-  template <Access accessType>
-  void CopyToDevice(Buffer<T, accessType> &other) {
+  template <Access accessType> void CopyToDevice(Buffer<T, accessType> &other) {
```

Comment:
```suggestion
  template <Access accessType>
  void CopyToDevice(Buffer<T, accessType> &other) {
```
Please leave this the way it was